pkg/ottl/e2e/e2e_test.go

```diff
@@ -1200,6 +1200,14 @@ func Test_e2e_converters(t *testing.T) {
 				tCtx.GetLogRecord().Attributes().PutInt("test", 2)
 			},
 		},
+		{
+			statement: `set(attributes["list"], Sort(Keys({"foo": "bar", "baz": "foo"})))`,
```

Comment:
Can we change this test and drop the `Sort` usage? It would be nice if we could make it use only the converter being tested (`Keys`).

Reply:
hey @edmocosta  the issue there is that maps do not guarantee the order and this becomes a flaky test failing form time to time